lib/packwerk/application_validator.rb

```diff
@@ -329,7 +329,8 @@ def package_glob
     end
 
     def package_manifests(glob_pattern)
-      PackageSet.package_paths(@configuration.root_path, glob_pattern).map(&:to_s)
+      PackageSet.package_paths(@configuration.root_path, glob_pattern)
+        .map { |f| File.realpath(f) }
```

Comment:
https://github.com/Shopify/packwerk/pull/38/files#diff-5ca9dcd54fd0b9eeef94e00ba6adc7822dc83e5dd2bfb9e8d96fd3c356de8b8aL328

Reply:
I verified that `File.realpath(Pathname)` returns a string, so there's no need to add `to_s` like in the old code.
```ruby
[9] pry(main)> File.realpath(Pathname.new("."))
=> "/Users/rochlefebvre/src/github.com/Shopify/parsnip"
```
👍

Reply:
\nit I'm thinking out loud here, but could it make sense to add a method to the PackageSet that returns all the realpaths for the given packages instead of ApplicationValidator being aware of retrieving that information? 🤔